src/main/java/edu/virginia/lib/fedora/aview/AView.java

```diff
@@ -135,4 +136,23 @@ public Response listFiles(@PathParam("accessionId") final String accessionId, @P
         filesTemplate.merge(context, w);
         return Response.ok().encoding("UTF-8").entity(w.toString()).build();
     }
-}
+/*
+	@GET
+	@Path("/{accessionId: [^/]*}/bags/{bagId: [^/]*}")
```

Comment:
If you're thinking you'd have the URL /accessions/id/bags/bagid/image-sets represent a view of all image sets that include images from the given bag, the path should be "/{accessionId: [^/]*}/bags/{bagId: [^/]*}/image-sets".

I'd probably put this method in the ImageSets class though since you'll want to use the code/logic that's already implemented in ImageSets.findImageSetsForBagId.